webpack.config.babel.js

```diff
@@ -434,8 +423,16 @@ const configNpmLib = () => ({
   ],
 })
 
+// Workaround for https://github.com/stephencookdev/speed-measure-webpack-plugin/issues/167
```

Comment:
Oh, I only just realized that this library was created by our lovely Stephen, former Onfido employee and SDK contributor :)